polylaue/model/series.py

```diff
@@ -337,6 +374,41 @@ def deserialize(self, d: dict):
         self.invalidate()
         super().deserialize(d)
 
-
-class ValidationError(Exception):
-    pass
+    # Editable fields
+    @classmethod
```

Comment:
I think this ought to be a `@staticmethod` instead of an `@classmethod`. For class methods, it's expected that the `cls` that is passed will be used to initialize and return an object of that class. Class methods are more like factory functions.